test/manipulation.test.js

```diff
@@ -1338,6 +1338,25 @@ describe('manipulation', function() {
         });
       }
     });
+    describe('replaceById', function() {
```

Comment:
Ah, it was my fault then, it's entirely possible I did not spot that `describe` was nested inside the `else` branch.

The new way is much cleaner 👍